### PR TITLE
Add and populate an active column on etl_shops

### PR DESCRIPTION
**Pull request by franckc**

### CODE REVIEW

File: backend/etl/processor.js [hunk 1]
```diff
@@ -130,10 +132,62 @@ class EtlJobProcessor {
...
-    // TODO(franck): figure out how to extract this information.
-    log.warn(`Shop ${shop.id}: custom domain extraction not implemented yet.`)
-    return null
+  // Check if the most recent publish (if any) used a custom domain.
+  async _customDomain(shop) {
+    const custom = await ShopDomain.findOne({
+      where: { shopId: shop.id },
+      order: [['id', 'desc']]
+    })
+    return custom ? custom.domain.indexOf('ogn.app') > 0 : false
```
Comment (mikeshultz) [inline backend/etl/processor.js:141]: I don't really think it has any benefit here, but you can use ILIKE like this with Sequelize:

    where: { domain: { [Op.iLike]: '%ogn.app' } }

No need to change.

File: backend/etl/processor.js [hunk 2]
```diff
@@ -130,10 +132,62 @@ class EtlJobProcessor {
...
+      data.numDiscounts !== prevData.numDiscounts
+    ) {
+      return true
+    }
+
+    // Check if the shop was deployed.
+    const published = await ShopDeployment.findOne({
+      where: {
+        shopId: shop.id,
+        createdAt: { [Sequelize.Op.gte]: prevData.createdAt }
```
Comment (mikeshultz) [inline backend/etl/processor.js:170]: Maybe you just want `Op.gt`?

**Review by mikeshultz — APPROVED** [review]
One comment, one question.  LGTM

File: backend/etl/processor.js (hunk 1 above)
Comment (franckc) [reply backend/etl/processor.js:141]: Ah yeah. Good to know. Thanks.

File: backend/etl/processor.js (hunk 2 above)
Comment (franckc) [reply backend/etl/processor.js:170]: Fixed.

**Pull request merged**